Gopkg.toml

```diff
@@ -108,7 +108,7 @@ required = [
 
 [[override]]
   name = "k8s.io/kubernetes"
-  version = "v1.15.4"
+  version = "=v1.15.4"
```

Comment:
dep wanted to use 1.15.6, so I locked the dependency; it will be upgraded to 1.16.3 via https://github.com/giantswarm/helmclient/pull/134